serverless.yml

```diff
@@ -10,8 +10,8 @@ plugins:
   - serverless-offline
 
 functions:
-  getDefects:
-    handler: handler.getTestTypes
+  getTestTypes:
+    handler: src/handler.getTestTypes
```

Comment:
Based on Temi's request, the function from "getTestTypes" to "handler"

Reply:
Done